### DISCUSSION TITLE
Update of Microsoft.CodeAnalysis

### DISCUSSION THREAD

**Discussion by Artromskiy** [AvaloniaUI/Avalonia#15404] [General]
Will there be an update to Microsoft.CodeAnalysis? In my project I am using version 4.10. Avalonia uses version 3.8, which was released back in 2020 and has language support up to version 9, while now the language is already at version 12

**Comment by maxkatz6** [comment]
It was completely removed from the Avalonia.Diagnostics project in 11.1-beta. So no point in updating it.

But in 11.0 it also isn't a big problem, as you can force specific version, by referencing nuget in your project.
